server/utils/constants.js

```diff
@@ -375,6 +376,35 @@ const DEVICE_FEATURE_TYPES = {
   SIGNAL: {
     QUALITY: 'integer',
   },
+  TELEVISION: {
+    BINARY: 'binary',
+    SOURCE: 'source',
+    GUIDE: 'guide',
+    MENU: 'menu',
+    TOOLS: 'tools',
+    INFO: 'info',
+    ENTER: 'enter',
+    RETURN: 'return',
+    EXIT: 'exit',
+    LEFT: 'left',
+    RIGHT: 'right',
+    UP: 'up',
+    DOWN: 'down',
+    CHANNEL_UP: 'channel-up',
+    CHANNEL_DOWN: 'channel-down',
+    CHANNEL_PREVIOUS: 'channel-previous',
+    CHANNEL: 'channel',
+    VOLUME_UP: 'volume-up',
+    VOLUME_DOWN: 'volume-down',
+    VOLUME_MUTE: 'volume-mute',
+    VOLUME: 'volume',
+    PLAY: 'play',
+    PAUSE: 'pause',
+    STOP: 'stop',
+    REWIND: 'rewind',
+    FORWARD: 'forward',
+    RECORD: 'record',
```

Comment:
How are all of those features handled? Not handled yet right?

Reply:
Those are not visible on dashboard, at least not yet, so not actionnable for now.
But incoming broadlink (IR remote) service will allow to use them.

For dashboard presentation, let's think about a nice display.

Reply:
Yes sure, just wanted to know if it was used or not :)